app/domain/audits/filter.rb

```diff
@@ -19,6 +21,15 @@ def audit_status=(value)
                       end
     end
 
+    def sort_by=(value)
+      if value.present?
```

Comment:
Perhaps we should have something a bit more robust here? The sort by value is only valid if it is a string containing an underscore followed by 'asc' or 'desc' aka `/\A[a-z]+_(asc|desc)\z/`. We've maybe overlooked that because at the moment we're concentrating on how this object is used by the controller upstream.

Reply:
Addressed in 499c54ad01373402047ea0225fd59545d5fc3f7d

Reply:
Addressed in 499c54ad01373402047ea0225fd59545d5fc3f7d